docs/topics/vx-runtime-info.adoc

```diff
@@ -0,0 +1,26 @@
+= {Vertx} Runtime Information
+
+.Certified {Vertx} Version
+
+{Vertx} version `3.42-3` is certified to run on OpenShift.
```

Comment:
This version number is most certainly wrong. One, it's `3.4.2`, not `3.42`. Two, it's something like `.redhat-N`, not `-3`

Looking at the CR1 Maven repo (available internally), the full version number is `3.4.2.redhat-006`.